core/templates/pages/exploration-player-page/services/extract-image-filenames-from-state.service.ts

```diff
@@ -106,7 +108,10 @@ export class ExtractImageFilenamesFromStateService {
           state.interaction.id === this.INTERACTION_TYPE_ITEM_SELECTION ||
           state.interaction.id === this.INTERACTION_TYPE_DRAG_AND_DROP_SORT) {
         let customizationArgsHtml = '';
-        state.interaction.customizationArgs.choices.value.forEach(
+        (<IMultipleChoiceInputCustomizationArgs |
```

Comment:
Please define a union type and use that instead.

Reply:
done